spec/models/offspring_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec.describe Offspring, type: :model do
+  it "should create a new instance given valid attributes" do
+    FactoryGirl.create(:offspring)
+  end
+
+  it "should initialize successfully as an instance of the described class" do
+    expect(subject).to be_a_kind_of described_class
+  end
+
+  it "should identify its parent class" do
```

Comment:
You don't need to test that STI is working.... That is Rails work. But it is quite close to what is needed.
You can test:
- That you can create a child class
- You can call it as the parent (so you can create an array of different children and act upon them)
- You delete them when you delete the parent
- Configuration is there (type of offspring by default, ages that are acceptable)
- The enum for course allows all elements (and not other values)